metadata_layer.py

```diff
@@ -98,14 +98,14 @@ def __init__(
         )
 
     @classmethod
-    def from_license_source(self, _db, **args):
+    def from_license_source(cls, _db, **args):
```

Comment:
Can these methods still work if they are defined as class methods but take in `self` as the first param?

Reply:
Yeah, the variable name is purely informative. But the previous code was wrong from an understandability viewpoint, and code like `self()` looks really off.